tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -445,14 +445,16 @@ def _enable_disable_volume_profiling(self):
             if volume.profiling_enabled in [None, ""]:
                 profiling_unknown_count += 1
         if profiling_unknown_count == len(volumes):
-            cluster.save()
             return
```

Comment:
Tere is no change is a cluster object from above statements, storing this without loading will leads to race condition, instead of keeping this i am remving this.

Tested

Reply:
Ack